static/scripts/util.js

```diff
@@ -0,0 +1,65 @@
+// Utility function to send an AJAX POST request
+
+function sendPostRequest(url, data, successCallback, errorCallback) {
+    $.ajax({
+        type: 'POST',
+        url: url,
+        data: data,
+        success: successCallback,
+        error: errorCallback
+    });
+}
+
+// Function to handle the change event of the yearSelect dropdown
+function handleYearSelectChange(params) {
+    $('#yearSelect').change(function () {
+        const selectedYear = $(this).val();
+        let currentEndpoint = params.current_endpoint;
+        let parts = currentEndpoint.split('/');
+
+        parts[parts.length - 1] = selectedYear; // Update the year in the endpoint
```

Comment:
I really think we should rather generate the `currentEndpoint` value from a call to `url_for` with a placeholder value for the year. This way if we change the route arguments position, this will be less prone to regressions.